doc/examples/plot_seam_carving.py

```diff
@@ -46,22 +46,22 @@
 
 out = transform.seam_carve(img, eimg, 'vertical', 200)
 plt.figure()
-plt.title('Resized using Seam-Carving')
+plt.title('Resized using Seam Carving')
 plt.imshow(out)
 
 """
 .. image:: PLOT2RST.current_figure
 
-As you can see, resizing as distorted the rocket and the objects around,
-whereas seam carving has reszied by removing the empty spaces in between.
+As you can see, resizing has distorted the rocket and the objects around,
```

Comment:
Feel free to make more sweeping changes - for example, this would probably be clearer still as "Resizing distorts the rocket and surrounding objects, whereas seam carving removes empty spaces and preserves object proportions."